components/properties/src/bidi_data.rs

```diff
@@ -97,27 +97,21 @@ impl<'a> BidiAuxiliaryPropertiesBorrowed<'a> {
     ///
     /// # Examples
     /// ```
-    /// use icu_properties::{bidi_data, bidi_data::BidiMirroringProperties};
+    /// use icu_properties::{bidi_data::bidi_auxiliary_properties, bidi_data::BidiMirroringProperties};
     ///
-    /// let data = bidi_data::load_bidi_auxiliary_properties_unstable(
-    ///     &icu_testdata::unstable(),
-    /// )
-    /// .expect("The data should be valid");
-    /// let bidi_data = data.as_borrowed();
```

Comment:
Nit: Reduce the diff with `let bidi_data = bidi_data::bidi_auxiliary_properties();`